resources/tags.yaml

```diff
@@ -0,0 +1,171 @@
+accounting:
+  title: 'Accounting'
+  description: 'Processing and communication of financial information'
+analytics:
+  title: 'Analytics'
+  description: 'Data or statistics to make intelligent marketing and business decisions'
+backend: 
+  title: 'Backend'
```

Comment:
I prefer 'Backend', if is an API, es expected that should be a service. The same for Database and DaaS. I think that we can avoid acronyms ever that we can.